openedx/adg/lms/mysite/models.py

```diff
@@ -0,0 +1,44 @@
+"""
+All models for mysite app
+"""
+from django.core.validators import FileExtensionValidator
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from model_utils.models import TimeStampedModel
+
+from .constants import ALLOWED_PROFILE_IMAGE_EXTENSIONS, MAX_COMMENT_TEXT_LENGTH, MAX_POST_TITLE_LENGTH
+
+
+class Post(TimeStampedModel):
+    """
+    Model for posts
+    """
+
+    title = models.CharField(max_length=MAX_POST_TITLE_LENGTH, unique=True, verbose_name=_('Post title'))
+    description = models.TextField(null=True, blank=True, verbose_name=_('Description'))
+    image = models.ImageField(
+        upload_to='team-members/images/', verbose_name=_('Image'),
+        validators=[FileExtensionValidator(ALLOWED_PROFILE_IMAGE_EXTENSIONS)],
+        help_text=_('You can add only png or jpg or jpeg or svg images'),
+        blank=True
+    )
+
+    class Meta:
+        verbose_name = _('Posts Comment')
+        app_label = 'mysite'
+
+    def __str__(self):
+        return f'{self.title}'
+
+
+class Comment(TimeStampedModel):
+    """
+    Model for post's comments
+    """
+
+    post = models.ForeignKey(Post, on_delete=models.CASCADE)
+    comment = models.CharField(max_length=MAX_COMMENT_TEXT_LENGTH, verbose_name=_('Comment'))
```

Comment:
`text` would be a better field name here. Similar model and field name can become confusing in further queries.